### PR TITLE
feat(proxy_to_vkr): update to cosmwasm 1.1

### PR DESCRIPTION
**Pull request by P-Yevhenii**

### CODE REVIEW

File: contracts/proxy_to_vkr/examples/schema.rs [hunk 1]
```diff
@@ -1,23 +1,11 @@
...
-    export_schema(&schema_for!(Cw20HookMsg), &out_dir);
-    export_schema(&schema_for!(QueryMsg), &out_dir);
-    export_schema(&schema_for!(MigrateMsg), &out_dir);
-    export_schema(&schema_for!(ConfigResponse), &out_dir);
-    export_schema(&schema_for!(CallbackMsg), &out_dir);
+    write_api! {
+        instantiate: InstantiateMsg,
+        query: QueryMsg,
+        execute: ExecuteMsg,
+        migrate: MigrateMsg,
```
Comment (ShadoySV) [inline contracts/proxy_to_vkr/examples/schema.rs:10]: This should be linked to owned migrate message

File: contracts/proxy_to_vkr/Cargo.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "generator-proxy-to-vkr"
-version = "0.0.0"
-authors = ["_astromartian"]
+version = "1.0.0"
+authors = ["Astroport"]
```
Comment (ShadoySV) [inline contracts/proxy_to_vkr/Cargo.toml]: Keep original authors

**Review by ShadoySV — CHANGES_REQUESTED** [review]

File: .github/workflows/code_coverage.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
+name: Code coverage
```
Comment (ShadoySV) [inline .github/workflows/code_coverage.yml]: We don't need code coverage here

File: .github/workflows/tests_and_checks.yml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        with:
+          profile: minimal
+          toolchain: 1.60.0
+          override: true
+          components: rustfmt, clippy
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9.7
```
Comment (ShadoySV) [inline .github/workflows/tests_and_checks.yml]: We don't need this

**Review by ShadoySV — CHANGES_REQUESTED** [review]

**Review by ShadoySV — APPROVED** [review]

File: .github/workflows/tests_and_checks.yml [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+            ~/.cargo/registry/cache
+            ~/.cargo/registry/index
+            target
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+
+      - name: Install stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: 1.60.0
```
Comment (epanchee) [inline .github/workflows/tests_and_checks.yml]: Lets use latest version: 1.64.0

File: contracts/proxy_to_vkr/Cargo.toml [hunk 6]
```diff
@@ -20,16 +20,13 @@ crate-type = ["cdylib", "rlib"]
...
-serde = { version = "1.0.125", default-features = false, features = ["derive"] }
-thiserror = { version = "1.0.24" }
-cw2 = "0.13.2"
-cw20 = "0.13.2"
+cosmwasm-std = "1.1"
+cw-storage-plus = "0.15"
+thiserror = { version = "1.0" }
+cw2 = "0.15"
+cw20 = "0.15"
+valkyrie = { git = "https://github.com/valkyrieprotocol/contracts.git", branch = "feature/migrate_terra2"  }
```
Comment (epanchee) [inline contracts/proxy_to_vkr/Cargo.toml]: I think better to pin exact version by commit hash, not by temporal branch

**Pull request merged**